webapp/app/[locale]/stake/_components/icons/hemi3xPointsIcon.tsx

```diff
@@ -0,0 +1,88 @@
+        fill="url(#paint1_linear_11619_12898)"
+      />
+    </g>
+    <path
+      d="M39.538 0.00242893C39.4556 -0.0123677 39.3781 0.0418866 39.3636 0.125734L38.3706 5.86189H37.6294L36.6364 0.125734C36.6219 0.0418866 36.5444 -0.0123677 36.462 0.00242893C32.9355 0.69787 30.2325 3.75584 30.0097 7.49445C30.0097 7.49938 30 7.65721 30 7.73613C30 7.74599 30 7.75586 30 7.76079C30 7.79038 30 7.81997 30 7.84957C30 7.85943 30 7.8693 30 7.88409C30 7.92355 30 7.95808 30 7.99753C30 11.963 32.7805 15.2627 36.4668 15.9926C36.5492 16.0074 36.6267 15.9532 36.6412 15.8693L37.6343 10.1332H38.3754L39.3636 15.8743C39.3781 15.9581 39.4556 16.0124 39.538 15.9976C43.0645 15.2972 45.7626 12.2392 45.9903 8.50062C45.9903 8.49569 46 8.33785 46 8.25894C46 8.24907 46 8.23921 46 8.23428C46 8.20468 46 8.17509 46 8.1455C46 8.13563 46 8.12577 46 8.11097C46 8.07152 46 8.03699 46 7.99753C46.0048 4.03204 43.2243 0.732395 39.538 0.00242893Z"
+      fill="white"
+    />
+    <defs>
+      <filter
+        color-interpolation-filters="sRGB"
```

Comment:
I think these will log warnings

```suggestion
        colorInterpolationFilters="sRGB"
```

See [docs](https://react.dev/learn/writing-markup-with-jsx#3-camelcase-salls-most-of-the-things)

Reply:
fixed